07-routing-gossip.md

```diff
@@ -80,13 +80,13 @@ announcement message; that is done by having a signature from each
     * [`64`:`node_signature_2`]
     * [`64`:`bitcoin_signature_1`]
     * [`64`:`bitcoin_signature_2`]
+    * [`2`:`len`]
```

Comment:
The problem with moving the features above the signatures, which indeed would allow us to swap out the signatures as well, was that then we'd need to have two serializations: one to create the signature and one for the on-wire transmission. We could either do that by skipping the signatures while signing, or mandating that the signatures should be zero-filled. I think both options are rather ugly.

If we really want to switch the messages this much we should probably just define a new message type altogether.